src/client/constants.h

```diff
@@ -0,0 +1,14 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef __CLIENT_CONSTANTS_H__
+#define __CLIENT_CONSTANTS_H__
+
+#include "core/constants.h"
+#include "crypto/constants.h"
```

Comment:
remove, `crypto/constants.h` has included in `core/constants.h`
```suggestion

```

Reply:
Done.